pkg/cmd/experimental/syncgroups/syncgroups.go

```diff
@@ -298,20 +330,22 @@ func (o *SyncGroupsOptions) Run() error {
 	}
 
 	// Now we run the Syncer and report any errors
-	syncErrors := syncer.Sync()
-	return kerrs.NewAggregate(syncErrors)
-}
+	if o.Confirm {
+		syncErrors := syncer.Sync()
```

Comment:
is there any kind of progress or status printed per group during the sync or as errors occur, or are all errors batched at the end?